packages/staking/src/features/Drawer/preferences/StepPreferencesContent.tsx

```diff
@@ -132,15 +133,17 @@ export const StepPreferencesContent = ({ popupView }: StepPreferencesContentProp
   return (
     <>
       <Flex flexDirection="column" gap="$32" alignItems="stretch">
-        <Box className={styles.delegationCardWrapper}>
-          <DelegationCard
-            balance={compactNumber(balancesBalance?.available?.coinBalance || '0')}
-            cardanoCoinSymbol={symbol}
-            distribution={displayData}
-            status={delegationStatus}
-            showDistribution
-          />
-        </Box>
+        {isSharedWallet && (
```

Comment:
Shouldn't it be with `!` instead?
```suggestion
        {!isSharedWallet && (
```

Reply:
resolved